go/pkg/digest/digest.go

```diff
@@ -156,9 +157,9 @@ func NewFromFile(path string) (Digest, error) {
 // It returns an error if there was a problem reading the file.
 func NewFromReader(r io.Reader) (Digest, error) {
 	h := HashFn.New()
-	buf := copyBufs.Get().([]byte)
+	buf := copyBufs.Get().(*[]byte)
```

Comment:
curious: what error do you see for this?

Reply:
https://staticcheck.io/docs/checks#SA6002

Reply:
Thanks.